test/test_modules_type_parser.rb

```diff
@@ -16,8 +17,6 @@ def examples
         "n"=>"n",
         "12 bomen"=>"12 bomen",
         "12 bomenasdfasdfsadf"=>"12 bomenasdfasdfsadf",
-        ""=>nil,
-        " "=>nil,
```

Comment:
I think leaving them as is might be better for some csv files. But lib your decision :)